packages/iota/src/validation/outputs/aliasOutput.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import bigInt from "big-integer";
+import { ALIAS_ADDRESS_TYPE } from "../../models/addresses/IAliasAddress";
+import { ISSUER_FEATURE_TYPE } from "../../models/features/IIssuerFeature";
+import { METADATA_FEATURE_TYPE } from "../../models/features/IMetadataFeature";
+import { SENDER_FEATURE_TYPE } from "../../models/features/ISenderFeature";
+import type { INodeInfoProtocol } from "../../models/info/INodeInfoProtocol";
+import { ALIAS_OUTPUT_TYPE, IAliasOutput } from "../../models/outputs/IAliasOutput";
+import { GOVERNOR_ADDRESS_UNLOCK_CONDITION_TYPE } from "../../models/unlockConditions/IGovernorAddressUnlockCondition";
+import { STATE_CONTROLLER_ADDRESS_UNLOCK_CONDITION_TYPE } from "../../models/unlockConditions/IStateControllerAddressUnlockCondition";
+import { validateFeatures, MAX_METADATA_LENGTH } from "../features/features";
+import { validateNativeTokens } from "../nativeTokens";
+import { IValidationResult, mergeValidationResults } from "../result";
+import { validateUnlockConditions } from "../unlockConditions/unlockConditions";
+import { validateAscendingOrder } from "../validationUtils";
+
+/**
+ * Zero alias id.
+ */
+const ZERO_ALIAS_ID = "0x0000000000000000000000000000000000000000000000000000000000000000";
+/**
+ * Maximum number of features that alias output can have.
+ */
+const MAX_ALIAS_FEATURES_COUNT = 2;
+/**
+ * Maximum number of immutable features that an nft output could have.
+ */
+const MAX_ALIAS_IMMUTABLE_FEATURES_COUNT = 2;
+/**
+ * Maximum number of unlock conditions that alias output can have.
+ */
+const MAX_ALIAS_UNLOCK_CONDITIONS_COUNT = 2;
```

Comment:
This is not really a "MAX" as we cannot have less that this number. This is **THE** count, we **must** have this number of unlock conditions defined.
* Rename to `ALIAS_UNLOCK_CONDITIONS_COUNT`
* Order the constants as they are ordered in the output: UnlockCondition, Feature, Immutable Feature